### PR TITLE
Initial introduction and getting started guides for ImageSharp.Drawing and Fonts

### PR DESCRIPTION
**Pull request by tocsoft**
Initial introduction and getting started guides for ImageSharp.Drawing and Fonts 

This adds the extra sections to the RC branch

### CODE REVIEW

File: articles/imagesharp.drawing/gettingstarted.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+>The official guide assumes intermediate level knowledge of C# and .NET. If you are totally new to .NET development, it might not be the best idea to jump right into a framework as your first step - grasp the basics then come back. Prior experience with other languages and frameworks helps, but is not required.
+
+### ImageSharp.Drawing - Paths and Polygons
+
+ImageSharp.Drawing provides several classes for build and manipulating various shapes and paths.
+
+- @"SixLabors.ImageSharp.Drawing.IPath" Root interface defining a path/polygon and the type that the rasterizer uses to generate pixel output.
+- This `SixLabors.ImageSharp.Drawing` namespace contains a variety of available polygons to speed up your drawing process.
+
+In addition to the vector manipulation APIs we also have the rasterization APIs that can convert your @"SixLabors.ImageSharp.Drawing.IPath"s to pixels.
```
Comment (JimBobSquarePants) [inline articles/imagesharp.drawing/gettingstarted.md]:
```suggestion
In addition to the vector manipulation APIs the library also contains rasterization APIs that can convert your @"SixLabors.ImageSharp.Drawing.IPath"s to pixels.
```

File: articles/fonts/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Built against [.NET Standard 1.3](https://docs.microsoft.com/en-us/dotnet/standard/net-standard), Fonts can be used in device, cloud, and embedded/IoT scenarios.  
+  
+### License  
+Fonts is licensed under under the terms of [GNU Affero General
+Public License, version 3](https://www.gnu.org/licenses/agpl-3.0.en.html). Commercial licensing options are available in addition to this license, see https://sixlabors.com/pricing for details.
+  
+### Installation
+  
+ImageSharp.Drawing is installed via [Nuget](https://www.nuget.org/packages/SixLabors.Fonts) with nightly builds available on [MyGet](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.Fonts).
```
Comment (JimBobSquarePants) [inline articles/fonts/index.md]:
```suggestion
Fonts is installed via [Nuget](https://www.nuget.org/packages/SixLabors.Fonts) with nightly builds available on [MyGet](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.Fonts).
```

File: articles/fonts/customrendering.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
+# Custom Rendering
+
+>[!WARNING]
+>Fonts is still considered BETA quality and we still reserve the rights to change the API shapes. We are yet to priorities performance in our font loading and layout APIs.
+
+>[!NOTE]
+>ImageSharp.Drawing already implements the glyph rendering for you unless you are rendering on other platforms we would recommend using the version build into that library.. this is a more advanced topic.
```
Comment (JimBobSquarePants) [inline articles/fonts/customrendering.md]:
```suggestion
>ImageSharp.Drawing already implements the glyph rendering for you unless you are rendering on other platforms we would recommend using the version provided by that library.. This is a more advanced topic.
```

File: articles/fonts/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+>[!WARNING]
+>Fonts is still considered BETA quality and we still reserve the rights to change the API shapes. WE are yet to priorities performance in our drawing APIs.
```
Comment (JimBobSquarePants) [inline articles/fonts/index.md]:
```suggestion
>Fonts is still considered BETA quality and we still reserve the rights to change the API shapes. We are yet to priorities performance in our drawing APIs.
```

File: articles/imagesharp.drawing/gettingstarted.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+### Drawing Polygons
+
+ImageSharp provides several options for drawing polygons weather you want to draw outlines or fill the shapes.
```
Comment (JimBobSquarePants) [inline articles/imagesharp.drawing/gettingstarted.md]:
```suggestion
ImageSharp provides several options for drawing polygons whether you want to draw outlines or fill shapes.
```

File: articles/imagesharp.drawing/index.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Introduction
+
+>[!WARNING]
+>ImageSharp.Drawing is still considered BETA quality and we still reserve the rights to change the API shapes. WE are yet to priorities performance in our drawing APIs.
```
Comment (JimBobSquarePants) [inline articles/imagesharp.drawing/index.md]:
```suggestion
>ImageSharp.Drawing is still considered BETA quality and we still reserve the rights to change the API shapes. We are yet to priorities performance in our drawing APIs.
```

File: articles/imagesharp.drawing/index.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+### What is ImageSharp.Drawing?
+ImageSharp.Drawing is a library build on top on ImageSharp to providing 2D Drawing extensions on top of ImageSharp.
```
Comment (JimBobSquarePants) [inline articles/imagesharp.drawing/index.md]:
```suggestion
ImageSharp.Drawing is a library build on top on ImageSharp to providing 2D Drawing extensions.
```

File: articles/imagesharp.drawing/index.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+ImageSharp.Drawing is designed from the ground up to be flexible and extensible. The library provides API endpoints for common vector and text processing operations adds the building blocks for building custom image.
```
Comment (JimBobSquarePants) [inline articles/imagesharp.drawing/index.md]:
```suggestion
ImageSharp.Drawing is designed from the ground up to be flexible and extensible. The library provides API endpoints for common vector and text processing operations adding the building blocks for building custom images.
```

File: docs/articles/ImageSharp/GettingStarted.html [hunk 9]
```diff
@@ -84,13 +84,13 @@ <h1 id="getting-started">Getting Started</h1>
 <h3 id="imagesharp-images">ImageSharp Images</h3>
 <p>ImageSharp provides several classes for storing pixel data:</p>
 <ul>
-<li><a class="xref" href="../../api/ImageSharp/SixLabors.ImageSharp.Image.html">Image</a> A pixel format agnostic image container used for general processing operations.</li>
-<li><a class="xref" href="../../api/ImageSharp/SixLabors.ImageSharp.Image-1.html">Image&lt;TPixel&gt;</a> A generic image container that allows per-pixel access.</li>
+<li>@&quot;SixLabors.ImageSharp.Image&quot; A pixel format agnostic image container used for general processing operations.</li>
```
Comment (JimBobSquarePants) [inline docs/articles/ImageSharp/GettingStarted.html:87]: 👍 Nice!

**Review by JimBobSquarePants — APPROVED** [review]
Great stuff! Just some very minor fixes and suggestions.

**Comment by tocsoft** [comment]
accepted them all

**Pull request merged**